file2.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE users(
+	username       TEXT PRIMARY KEY NOT NULL,
+	email          TEXT    NOT NULL,
+	password       TEXT    NULLABLE,
+	age            INT     NULLABLE,
+	admin          BOOLEAN NULLABLE,
+	country        TEXT    NULLABLE,
```

Comment:
Is `country` meant to be like a user-entered location (completely-free form, like maybe they say "mars")? If so, I think we should rename to something like `location`, if not I think we should something like an enum of ISO codes or FK to a `countries` table